partitura/utils/music.py

```diff
@@ -1217,16 +1350,17 @@ def note_array_from_part_list(part_list, unique_id_per_part=True,
     onset_unit, _ = get_time_units_from_note_array(note_array)
 
     # sort by onset and pitch
-    pitch_sort_idx = np.argsort(note_array['pitch'])
+    pitch_sort_idx = np.argsort(note_array["pitch"])
     note_array = note_array[pitch_sort_idx]
-    onset_sort_idx = np.argsort(note_array[onset_unit], kind='mergesort')
+    onset_sort_idx = np.argsort(note_array[onset_unit], kind="mergesort")
     note_array = note_array[onset_sort_idx]
 
     return note_array
 
 
-def slice_notearray_by_time(note_array, start_time, end_time, time_unit='auto',
-                            clip_onset_duration=True):
+def slice_notearray_by_time(
```

Comment:
As in similar comments, I slightly prefer the old formatting of definitions, but I am ok with the new one if that is preferred by PEP8.

Reply:
Fixed the awkward line splitting in the method definition (to be consistent with other re-formatted definitions) in commit 1724ae1.